zenpacklib.py

```diff
@@ -621,9 +624,21 @@ def _create_indexes(cls, zcatalog, spec):
                 # reindex all objects of this type so they are added to the
                 # catalog.
                 results = ICatalogTool(context).search(types=(classname,))
+
```

Comment:
Dave, I left that commented code in because I wanted to see whether the catalog retrieval failure should be dealt with then and there (by uncataloging) or just reported for later action.  What do you think?

I could also uncomment it, but add a method argument like "leaveObjects" defaulted to False (reporting only)